.travis.yml

```diff
@@ -187,7 +187,7 @@ jobs:
     env:
     - DOCKER_REPOSITORY=geosx/centos7.6.1810-gcc8.3.1-cuda10.1.243
     - CMAKE_BUILD_TYPE=Release
-    - BUILD_AND_TEST_ARGS="--disable-unit-tests --reduce-install-logs --disable-schema-deployment"
+    - BUILD_AND_TEST_ARGS="--disable-unit-tests --disable-schema-deployment"
```

Comment:
I think you forgot to remove https://github.com/GEOSX/GEOSX/blob/b6b0e875d1b55bca361497bb572ef83a25c31f65/scripts/travis_build_and_test.sh#L83-L87 which lead me to `--reduce-install-logs` being dummy.

Reply:
yes